.evergreen/scripts/abi-compliance-check-test.sh

```diff
@@ -105,21 +102,53 @@ DOC
 <skip_headers>
   /v_noabi/
 </skip_headers>
+
+<skip_namespaces>
+  bsoncxx::detail
+  mongocxx::detail
```

Comment:
To confirm: is skipping `bsoncxx::v_noabi` unnecessary? I expect "yes" since `v1` headers will not include `v_noabi` headers and `/v_noabi` is already included in `skip_headers`.

Reply:
Correct: unlike `v_noabi` headers which include `v1`, `v1` headers do not include `v_noabi` headers; the `bsoncxx::v_noabi` namespace is completely absent when only `v1` headers are included.